Makefile

```diff
@@ -95,9 +95,18 @@ help-extended: #HELP Display extended help.
 #SECTION Development
 
 .PHONY: lint
-lint: $(GOLANGCI_LINT) #HELP Run golangci linter.
+lint: lint-custom $(GOLANGCI_LINT) #HELP Run golangci linter.
 	$(GOLANGCI_LINT) run --build-tags $(GO_BUILD_TAGS) $(GOLANGCI_LINT_ARGS)
 
+.PHONY: custom-linter-build
+LINTER_DIR := ./hack/ci/custom-linters/cmd
+custom-linter-build: # HELP Build custom linter
+	cd $(LINTER_DIR) && go build -tags '$(GO_BUILD_TAGS)' -o $(ROOT_DIR)/bin/custom-linter
```

Comment:
```suggestion
.PHONY: custom-linter-build
custom-linter-build: # HELP Build custom linter
	go build -tags '$(GO_BUILD_TAGS)' -o ./bin/custom-linter ./hack/ci/custom-linters/cmd
```

Reply:
The problem was with the go.mod 
Now that is solved. 
Thank you.